docs/pyboard/tutorial/script.rst

```diff
@@ -43,7 +43,7 @@ have as to what happens next:
 Okay, so you should now have the pyboard connected as a USB flash drive, and
 a window (or command line) should be showing the files on the pyboard drive.
 
-The drive you are looking at is known as ``/flash`` by the pyboard, and should contain
+The drive you are looking at is known as ``/PYBFLASH`` by the pyboard, and should contain
```

Comment:
Actually, I think it's correct the way it was: doing `os.listdir('/')` on the pyboard shows that the flash is mounted as the directory `/flash`.

Reply:
Oh! I misread that

Reply:
Yeah, it is a bit cryptic.  Maybe there could be an extra sentence here describing what I said above.

Reply:
I just removed that change on the line (I hadn't made partial git reverts before). Should I add a new sentence to clarify into this PR, add an Issue, or just add it to the doc I was writing my notes in while going through the tutorial (and where I'll be making PRs from)?

Reply:
Since the change would be a doc improvement, it would fit better in your notes.